app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,11 @@
+from abc import ABC
+from app.players.player import Player
```

Comment:
Do not forget to separate different groups of imports with a blank line according to PEP-8.